### PR TITLE
node 19.8.1

### PR DESCRIPTION
**Pull request by Wallunen**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

File: Formula/node.rb [hunk 1]
```diff
@@ -49,8 +49,8 @@ class Node < Formula
   # We track major/minor from upstream Node releases.
   # We will accept *important* npm patch releases when necessary.
   resource "npm" do
-    url "https://registry.npmjs.org/npm/-/npm-9.5.1.tgz"
-    sha256 "f5f0cbf0d054e604bf9497b94f095f892702e9259361a6577b0a0fd49755cfbf"
+    url "https://registry.npmjs.org/npm/-/npm-9.6.2.tgz"
+    sha256 "45c1ec549d2a96b3980242e2105fc2d61cc9872fc6e83c97a609ac4c7d123e55"
```
Comment (carlocab) [inline Formula/node.rb:53]: Node 19.8.1 still ships npm 9.5.1. Please revert this change.

**Review by carlocab — DISMISSED** [review]

**Review by p-linnane — APPROVED** [review]
